src/cdk/scrolling/virtual-scroll-viewport.html

```diff
@@ -9,4 +9,6 @@
   Spacer used to force the scrolling container to the correct size for the *total* number of items
   so that the scrollbar captures the size of the entire data set.
 -->
-<div class="cdk-virtual-scroll-spacer" [style.transform]="_totalContentSizeTransform"></div>
+<div class="cdk-virtual-scroll-spacer"
+     [style.width]="orientation == 'horizontal' ? _totalContentSize + 'px' : ''"
```

Comment:
nit: it might be micro-optimizing a bit, but would it make sense to have this logic inside the component when setting `_totalContentSize`? Otherwise Angular will run the ternary and the string concatenation on each change detection run.

Reply:
yeah, probably a good idea